db/notifications.go

```diff
@@ -0,0 +1,64 @@
+package db
+
+import (
+	"encoding/json"
+
+	"github.com/flanksource/duty/models"
+	v1 "github.com/flanksource/incident-commander/api/v1"
+	"github.com/google/uuid"
+)
+
+func PersistNotificationFromCRD(obj *v1.Notification) error {
```

Comment:
@moshloop For all the CRD implemented resources, we have a `spec` column in the corresponding database table. However, notification is already built in a different way. We don't have a `spec` column for it.

Instead of modifying the notification table by removing existing columns and adding the `spec` column, I've implemented it this way. So far I don't really see why this would be an issue.